src/Symfony/Component/HttpKernel/Tests/Controller/LegacyArgumentResolverTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Controller;
+
+use Symfony\Component\HttpKernel\Controller\LegacyArgumentResolver;
+use Symfony\Component\HttpKernel\Controller\ControllerResolver;
+use Symfony\Component\HttpKernel\Tests\Fixtures\Controller\VariadicController;
+use Symfony\Component\HttpFoundation\Request;
+
+class LegacyArgumentResolverTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetArguments()
```

Comment:
Please split this big test in multiple smaller tests instead. This way, a failing case for the first test does not prevent running other cases